node/init_test.go

```diff
@@ -25,3 +25,59 @@ func TestInitLight(t *testing.T) {
 	ok := IsInit(dir, Light)
 	assert.True(t, ok)
 }
+
+func TestInitErrForInvalidPath(t *testing.T) {
+	path := "/invalid_path"
+	errLight := Init(path, Light)
+	errFull := Init(path, Full)
+
+	require.Error(t, errLight)
+	require.Error(t, errFull)
```

Comment:
It's not a good idea to have 1 testing 2 cases as this can cause confusion for future contributors as the code base grows rapidly over time. 

- ~~Parametrised(java,python,c++ flashbacks incoming)~~ [Table driven tests](https://dave.cheney.net/2019/05/07/prefer-table-driven-tests) can fix this or 
- either completely separating to 2 kind-a-like the same scenario based cases

Reply:
Hi @Bidon15 , thanks for the hint. I have defined the common behaviour for each test and then run it for both nodes since test cases are doing the same for Light and Full nodes.